webpack/assets/javascripts/react_app/components/common/forms/FormAutocomplete/FormAutocomplete.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormGroup, Col, ControlLabel } from 'patternfly-react';
+import AutoComplete from '../../../AutoComplete';
+import { noop } from '../../../../common/helpers';
+
+const FormAutocomplete = ({
+  controller,
+  searchQuery,
+  error,
+  results,
+  status,
+  resetData,
+  getResults,
+  initialUpdate,
+  url,
+  isDisabled,
+  trigger,
+  data,
+}) => (
+  <FormGroup className="form-autocomplete">
+    <Col componentClass={ControlLabel} md={2}>
+      Search
```

Comment:
need to be translated
and I think the label should be changeable, the consumer may use another label, right?

Reply:
fixed